src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -185,15 +215,16 @@ public void Save(ModelSaveContext ctx)
 
         private const Mkl.Layout Layout = Mkl.Layout.RowMajor;
 
-        // Stores whitening matrix as Float[] for each column.
-        private readonly Float[][] _models;
-        // Stores inverse ("recover") matrix as Float[] for each column. Temporarily internal as it's used in unit test.
+        // Stores whitening matrix as float[] for each column. _models[i] is the whitening matrix of the i-th input column.
+        private readonly float[][] _models;
+        // Stores inverse ("recover") matrix as float[] for each column. Temporarily internal as it's used in unit test.
         // REVIEW: It doesn't look like this is used by non-test code. Should it be saved at all?
-        internal readonly Float[][] InvModels;
+        private readonly float[][] _invModels;
 
         internal const string Summary = "Apply PCA or ZCA whitening algorithm to the input.";
 
-        public const string LoaderSignature = "WhiteningTransform";
+        internal const string FriendlyName = "Whitening Transform";
+        internal const string LoaderSignature = "VectorWhiteningTransform";
```

Comment:
>VectorWhiteningTransform [](start = 49, length = 24)

Why you changing LoaderSignature?
I think this would screw up loading old models.